docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,312 @@ sidebar_position: 18
 
-```
-tar xvf uid2-operator.tar.gz
-cd uid2-operator
-occlum run /bin/launcher
-```
+    Parameter | Description |
+   | :--- | :--- |
+   | `operatorIdentifier` | The name of the managed identity that will launch the container.<br/>Default: `uid-operator`. |
+   | `operatorKeyName` | The operator key secret name.<br/>Default: `operator-key`. |
+
+3. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vault --resource-group {RESOURCE_GROUP_NAME} --parameters vault.parameters.json  --template-file vault.json
+   ```
+
+### Set Up the VPC Network
+
+The next step is to set up the VPC network.
+
+The following diagram illustrates the virtual private cloud that hosts a UID2 Private Operator in Microsoft Azure.
+
+![VPC Network](images/operator-azure-drawio.png)
+
+Follow these steps:
+
+1. (Optional) If you don't want to accept the defaults, update the `vnet.parameters.json` file with the following values. These parameters have default values and in most cases you won't need to make any updates.
+
+    Parameter | Description |
+   | :--- | :--- |
+   | `vnetName` | The virtual network name.<br/>Default: `unified-id-network` |
+   | `computeSubnetName` | The name of the subnet that runs the UID2 Operator.<br/>Default: `unified-id-subnet-operators` |
+   | `gatewaySubnetName` | The name of the subnet that runs the UID2 Gateway.<br/>Default: `unified-id-subnet-gateway` |
+   | `VnetAddressPrefix` | The vnet address prefix.<br/>Default: `10.0.0.0/20` |
+   | `computeSubnetPrefix` | The vnet address prefix of the subnet that is delegated to run the UID2 Operator.<br/>Default: `10.0.0.0/24` |
+   | `gatewaySubnetPrefix` | The vnet address prefix of the subnet that runs the UID2 Gateway.<br/>Default: `10.0.0.0/28` |
+
+2. Run the following command to trigger the deployment:
+
+   ```
+   az deployment group create --name vnet --resource-group {RESOURCE_GROUP_NAME} --parameters vnet.parameters.json  --template-file vnet.json
+   ```
+
+### Complete the UID2 Private Operator Setup
+
+The next step is to bring up multiple Azure Container Instances (ACIs) in the VPC subnetwork that you created.
+
+Follow these steps:
+
+1. Update the `operator.parameters.json` file with the following required values:
+
+   | Parameter | Description |
+   | :--- | :--- |
+   | `vaultName` | The name of the key vault for hosting the operator key secret. The value must match the name you created in [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup). |
+   | `operatorKeyName` | The operator key secret name. The value must match the value specified in [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup). If you accepted the default, the value is `operator-key`. |
```

Comment:
Move this to optional params table to be consistent? In above `Complete Key Vault and Managed Identity Setup` section this is in optional params table, so if user use the default value in above section, they don't need to update the value here.

Reply:
@yishi-ttd :  This is not a question I could answer. In the source doc it's marked as Required and, although there is a default in vault.parameters.json, I don't know that there is a default here. Anyway moving it based on your comment.
I'm presuming you mean just operatorKeyName since vaultName is required in the earlier file.

Reply:
Done